### ISSUE TITLE
Accessibility setting - user should be able to adjust the sound of thunder in the settings

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#2837]
some people with hyper-activity disorders may experience negative effects from the sound being too loud (as was reported by one user on discord) as well as some people may want to lower down thunder effects specifically without sacrificing other weather effects.

Possibly the same setting should apply collectively also to dynamite explosions and close-range gunshots?

**Comment by Leiaviathan** [comment]
Yes, please! I really struggle with some of the noises in-game being so loud they cause over stimulation. 
The rain is louder inside of some of the buildings (like One-Stop-Shop), machines can be extremely loud, opening and closing containers repeatedly, walking on loud surfaces. It's hard to explain to people who don't experience these problems, but the noise can make me unable to focus on what I'm doing, cause stress and even headache. 
Turning off the sound completely or lowering volume overall helps, but it also makes the game a lot less fun to play, and I often rely on audio clues like when waiting for machines to finish.
